cita-network/src/node_manager.rs

```diff
@@ -106,22 +106,57 @@ impl NodesManager {
         if let Some(enable_tls) = cfg.enable_tls {
             node_mgr.enable_tls = enable_tls;
         }
-        if let Some(known_addrs) = cfg.peers {
-            for addr in known_addrs {
+        if let Some(cfg_addrs) = cfg.peers {
+            for addr in cfg_addrs {
                 if let (Some(ip), Some(port)) = (addr.ip, addr.port) {
                     let addr_str = format!("{}:{}", ip, port);
-                    node_mgr.config_addrs.insert(addr_str, false);
+                    node_mgr.config_addrs.insert(addr_str, None);
                 } else {
                     warn!("[NodeManager] ip(host) & port 'MUST' be set in peers.");
                 }
             }
         } else {
             warn!("[NodeManager] Does not set any peers in config file!");
         }
-
         node_mgr
     }
 
+    pub fn notify_config_change(
+        rx: StdReceiver<DebouncedEvent>,
+        node_client: NodesManagerClient,
+        fname: String,
+    ) {
+        loop {
+            match rx.recv() {
```

Comment:
Nesting too deep.
Maybe use `if let` and refactor this function?

Reply:
I didn't know what you mean ? if let can reduce nest ?

Reply:
Just an advice.
Check this:

![Screenshot from 2019-04-28 17-39-22](https://user-images.githubusercontent.com/8768261/56862414-39356400-69dd-11e9-9af9-b1a165e8f761.png)